### DISCUSSION TITLE
Integration with Rebound 11

### DISCUSSION THREAD

**Discussion by IviriusMain** [character-map-uwp/Character-Map-UWP#363] [Q&A] [answered]
I would like to integrate Character Map UWP with Rebound 11. My only request is to be allowed to download Character Map UWP from Rebound Hub in the background when installing. I can also leave a link to the Microsoft Store page of the app for users to check it out directly.

**Comment by JohnnyWestlake** [comment, marked as answer]
This is fine with me;
